### PR TITLE
Fixed editable datagrid cell popover, made inactive collection buttons disabled

### LINKED ISSUE #220: Add disabled attribute to add/remove item buttons in collection that don't allow these operations
Currently even if you specify `allow_add = false` and/or `allow_delete = false` the buttons are still created, but inactive. They should not be rendered at all.

### PR DESCRIPTION
**Pull request by szymach**
Resolves #220 and #221. Also made the popover wider

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open open for review once (if) tests pass

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

File: Resources/views/Form/form_theme.html.twig [hunk 1]
```diff
@@ -32,8 +35,9 @@
             {{ form_widget(form) }}
             {{ form_errors(form) }}
         </div>
+        {% set disabled = attr['data-allow-delete'] ? '' : 'disabled ' %}
         <div class="col-sm-1">
-            <a class="pull-right btn btn-danger collection-remove" title="{{ 'crud.form.collection.remove'|trans({}, 'FSiAdminBundle') }}"><i class="glyphicon glyphicon-trash"></i></a>
+            <a class="{{ disabledAdd }}pull-right btn btn-danger collection-remove" title="{{ 'crud.form.collection.remove'|trans({}, 'FSiAdminBundle') }}"><i class="glyphicon glyphicon-trash"></i></a>
```
Comment (chives) [inline Resources/views/Form/form_theme.html.twig]: `disabledAdd` -> `disabled`

File: Behat/Context/AdminContext.php [hunk 2]
```diff
@@ -258,6 +268,22 @@ public function collectionShouldHaveButton(NodeElement $collection, $buttonName)
     }
 
     /**
+     * @Then /^all buttons for adding and removing items in (non-editable collection "[^"]*") should be disabled$/
+     */
+    public function allCollectionButtonsDisabled(NodeElement $collection)
```
Comment (szymach) [inline Behat/Context/AdminContext.php]: needs fixing

File: Resources/views/Form/form_theme.html.twig [hunk 3]
```diff
@@ -32,8 +35,9 @@
             {{ form_widget(form) }}
             {{ form_errors(form) }}
         </div>
+        {% set disabled = attr['data-allow-delete'] ? '' : 'disabled ' %}
         <div class="col-sm-1">
-            <a class="pull-right btn btn-danger collection-remove" title="{{ 'crud.form.collection.remove'|trans({}, 'FSiAdminBundle') }}"><i class="glyphicon glyphicon-trash"></i></a>
+            <a class="{{ disabled }}pull-right btn btn-danger collection-remove" title="{{ 'crud.form.collection.remove'|trans({}, 'FSiAdminBundle') }}"><i class="glyphicon glyphicon-trash"></i></a>
```
Comment (norzechowicz) [inline Resources/views/Form/form_theme.html.twig]: just put that condtion here instead of creating extra variable.

**Comment by norzechowicz** [comment]
looks good, good job. Could you please post here screenshot of how editable popup looks now?

File: Resources/views/Form/form_theme.html.twig (hunk 3 above)
Comment (szymach) [inline Resources/views/Form/form_theme.html.twig]: the variable is ok, the problem is that the behat step does not find the collection buttons

Comment (norzechowicz) [inline Resources/views/Form/form_theme.html.twig]: no its not ok, its extra useless variable which needs to be compiled by twig to php. If there is no reason to keep it please remove it.

**Comment by szymach** [comment]
Like this popup

**Comment by norzechowicz** [comment]
can u also post screen about how it looks for mutiple mapped fields?

**Comment by szymach** [comment]
I took the liberty of adding labels to multiple mapped fields (only them), so you know which fields you are editing. popup

**Comment by norzechowicz** [comment]
awesome!

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Pull request merged**